packages/devtools_shared/lib/src/memory/memory_json.dart

```diff
@@ -341,3 +341,15 @@ class AllocationMemoryJson extends MemoryJson<ClassHeapStats> {
       '"${MemoryJson.jsonDevToolsScreenField}": "${MemoryJson.devToolsScreenValueMemory}", '
       '"${MemoryJson.jsonDataField}": [\n';
 }
+
+extension type _OldData(Map<String, dynamic> data) {
+  Map<String, dynamic> get class_ => data['class'];
+
+  int get bytesCurrent => data['bytesCurrent'];
+
+  int get bytesDelta => data['bytesDelta'];
+
+  int get instancesCurrent => data['instancesCurrent'];
+
+  int get instancesDelta => data['instancesDelta'];
```

Comment:
Do you want to add the casts here in a step towards enabling `strict-casts`?

Reply:
Not particularly 🤣 

I'm more in favor of "small changes" for this work, but adding the casts _now_ would highlight how the code will ultimately look, if trying to avoid casts-from-dynamic, _and_ all dynamic calls.